README.md

```diff
@@ -9,17 +9,46 @@ The SUI-Card component features a placeholder to portray a custom primary and a
 
 ## Usage
 
-The **SUI-Card** allows to pass custom content in the primary placeholder and also provides a `sui-Card-primary` class as a wrapper.
+The **SUI-Card** allows to pass custom content in the primary placeholder and also provides a `sui-Card-primary` class as a wrapper. This primary content **is required**.
+
+Secondary content might be passed or not to SuiCard component. If content is provided will be rendered inside a generic *div* `sui-Card-secondary` wrapper.
+If no content is provided this contained will not be rendered at all.
 
 Check out this example below
 ```javascript
 
 ReactDom.render(
     <SuiCard
-      primary={
-        <SuiMultimedia images={data} />
+      primary={[
+        <a href={data.link}><img src={data.src} /></a>
```

Comment:
I'm going to add a new use case to show how to implement a SuiMultimedia component